app/frontend/src/submissions/components/SubmissionForm/Yield.vue

```diff
@@ -120,17 +111,15 @@ import inputBindingsMixin from '@/common/inputBindingsMixin.js'
 export default {
   mixins: [inputBindingsMixin],
   props: {
-    productionData: {
-      type: Array,
-      default: () => ([])
-    },
     yieldEstimationMethod: String,
     yieldEstimationRate: String,
     yieldEstimationDuration: String,
     staticLevel: String,
     drawdown: String,
     hydroFracturingPerformed: null,
     hydroFracturingYieldIncrease: String,
+    recommendedPumpDepth: Number,
+    recommendedPumpRate: Number,
```

Comment:
This might throw errors/warning once values get inputted, since they will be a string by default.